packages/core/src/dropdown/dropdown.stories.mdx

```diff
@@ -4,13 +4,6 @@ import { Meta, Props, Story, Preview } from '@web/storybook-prebuilt/addon-docs/
 
 # Dropdowns
 
-<cds-alert-group status="warning" cds-layout="m-b:lg">
```

Comment:
these components were only previously available in the `next` version - should they go through a beta period in the main package as well?

Car we should certainly be removing from beta, but I'm not sure about dropdown?

Reply:
ah sorry, I thought we had that in v5. I'll get the beta added back to dropdown

Reply:
reverted